ios/brave-ios/Sources/Brave/WebFilters/ShieldStats/Adblock/CachedAdBlockEngine.swift

```diff
@@ -134,7 +134,7 @@ public class CachedAdBlockEngine {
   }
 
   /// This returns all the user script types for the given frame
-  @MainActor func makeEngineScriptTypes(
+  func makeEngineScriptTypes(
```

Comment:
Does this need to be done on the main thread specifically? All methods called on an `actor` will happen on the cooperative concurrency thread pool

Reply:
no i don't believe it does. This main actor is just some leftover code when there was some main actor calls that were needed in the past.